tests/release/conftest.py

```diff
@@ -90,11 +104,48 @@ def release_output_dir() -> Path:
 
 
 @pytest.fixture(scope="session")
```

Comment:
This being session-scoped through me off. I think it's ok but let me talk it out (silly since I designed it, I know 🤷 ).

All of these for-release tests will be launched by passing in parameters to the cli, e.g. `pytest -m release --dataset xxx --population yyy ...`. That means for the entire session, regardless of how many tests are actually marked as `release`, we want to run specificall for the parameters passed in (dataset, population, etc).

What we've given up by doing this is the ability to group parameters. Right? Like, we can't have a single pytest call test both census _and_ wic datasets (for example). And if we ever decide we _do_ want to be able to do that then these fixtures will have to all change scope.